<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zugbuchung</class>
 <widget class="QDialog" name="zugbuchung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zugbuchung</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IDlineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Abfahrt:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="FromdateTimeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ankunft:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="TodateTimeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Preis:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PricelineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Von:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FromlineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Nach:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TolineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Ãœber:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="connectingStextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
